kafka-ui-api/pom.xml

```diff
@@ -436,6 +436,82 @@
                 </plugins>
             </build>
         </profile>
+        <profile>
```

Comment:
If I understood it right it is just a copy of existing prod profile with some exclusions. If so - we'd better use plugin.configuration.skip property without any duplication, like
<img width="631" alt="Screenshot 2022-03-20 at 14 29 43" src="https://user-images.githubusercontent.com/702205/159160128-4daa80dc-fc0a-41fe-a321-72bb397c23ed.png">